internal/clearingway/handlers.go

```diff
@@ -376,6 +389,81 @@ func (c *Clearingway) Roles(s *discordgo.Session, i *discordgo.InteractionCreate
 	}
 }
 
+func (c *Clearingway) RemoveAll(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	g, ok := c.Guilds.Guilds[i.GuildID]
+	if !ok {
+		fmt.Printf("Interaction received from guild %s with no configuration!\n", i.GuildID)
+		return
+	}
+
+	// Ignore messages not on the correct channel
+	if i.ChannelID != g.ChannelId {
+		fmt.Printf("Ignoring message not in channel %s.\n", g.ChannelId)
+	}
+
+	err := discord.StartInteraction(s, i.Interaction, "Removing all Clearingway related roles...")
+	if err != nil {
+		fmt.Printf("Error sending Discord message: %v\n", err)
+		return
+	}
+
+	member, err := c.Discord.Session.GuildMember(g.Id, i.Member.User.ID)
+	if err != nil {
+		err = discord.ContinueInteraction(s, i.Interaction, err.Error())
+		if err != nil {
+			fmt.Printf("Error sending Discord message: %v\n", err)
+		}
+		return
+	}
+
+	clearingwayRoles := []*Role{}
+	for _, r := range g.AllRoles() {
+		if r.Skip {
+			continue
+		}
+		clearingwayRoles = append(clearingwayRoles, r)
+	}
```

Comment:
this might seem straight forward but let's add a comment for first time readers on what we're trying to accomplish with this code block and how we'll use it in the later code